app/components/SearchMovieCards/SearchMovieCards.types.ts

```diff
@@ -0,0 +1,9 @@
+export type SearchMovieCardsType = {
+  lists: Array<{
+    searchImage: { imgSrc: string; altText: string };
+    title: string;
+    date: string;
+    description: string;
```

Comment:
These props should be set for SearchMovieCard.
```suggestion
   image: { imgSrc: string; altText: string };
    title: string;
    date: string;
    description: string;
```